src/main/java/org/opentripplanner/updater/alerts/AlertsUpdateHandler.java

```diff
@@ -95,6 +95,12 @@ private void handleAlert(String id, GtfsRealtime.Alert alert) {
             if (informed.hasRouteId()) {
                 routeId = informed.getRouteId();
             }
+
+            String direction = null;
+            if (informed.hasTrip() && informed.getTrip().hasDirectionId()) {
+                direction = String.valueOf(informed.getTrip().getDirectionId());
```

Comment:
Given that OBA is on the way out and the GTFS Reference defines it as a "binary value", I'd prefer to just stick with what GTFS-RT tells us.

But I should probably stop nitpicking your pull request by now, sorry! ;)